cleanlab/filter.py

```diff
@@ -14,15 +14,13 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with cleanlab.  If not, see <https://www.gnu.org/licenses/>.
 
-
-"""Filter (previously in Cleanlab 1.0, this module was called Pruning)
-
-Contains methods for estimating the latent indices of all label issues.
-This code uses advanced multiprocessing to speed up computation.
-see: https://research.wmz.ninja/articles/2018/03/ (link continued below)
-on-sharing-large-arrays-when-using-pythons-multiprocessing.html
 """
+Methods to identify which examples have label issues. 
+Some of this code uses advanced multiprocessing to speed up computation: 
+https://research.wmz.ninja/articles/2018/03/on-sharing-large-arrays-when-using-pythons-multiprocessing.html
 
+Note: Previously in cleanlab versions <= 1.0, this module was called `pruning`.
```

Comment:
Does `v1.0` here need to be more specific to `v1.0.1`?

Reply:
updated